js/classes/entities/foreground/hud.js

```diff
@@ -121,15 +133,17 @@ class Hud {
         // if the user clicks on the pause/play button, toggle GAME.running
         if (GAME.user.firing && this.pauseButton.isMouseOver()) {
             GAME.running = !GAME.running;
+            ASSET_MGR.playAudio(SFX.UI_HIGH_BEEP.path, SFX.UI_HIGH_BEEP.volume);
             // TODO: open options menu here
+
         }
     }
 
     /**
      * Draw any HUD elements not drawn by individual components.
      */
     draw() {
-        this.chadHead.drawFrame(new Vector(10, 10), Chad.SCALE);
+        this.chadHead.drawFrame(new Vector(10, 10), CHAD.scale);
```

Comment:
I'm honestly not sure but do we actually want to scale Chad's head icon in the HUD when he uses power ups?